cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,61 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+import Long from 'long'
+import chalk from 'chalk'
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.integer({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+    isVerified: flags.boolean({
+      required: true,
+      description: 'Verification state of the validator',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId, isVerified } = this.parse(VerifyValidatorAccountCommand).flags
+
+    const id = Long.fromNumber(memberId)
+    const meta = new RemarkMetadataAction({
+      verifyValidator: new VerifyValidator({
+        memberId: id,
+        isVerified,
+      }),
+    })
+
+    const message = metadataToString(RemarkMetadataAction, meta)
+
+    const nowGroup = this.group
+
+    await this.setPreservedState({ defaultWorkingGroup: WorkingGroups.Membership })
```

Comment:
Hey @zeeshanakram3 it looks to me like the code from this PR should work. I'm just wondering whether these 2 lines and the line 59 are necessary.

Could you have another look please ? This is blocking the validator dashboard QA

Reply:
Right, we can actually specify the working group using `--group` flag, so this is not necessary at all. Please remove these